neonvm/apis/neonvm/v1/virtualmachine_types.go

```diff
@@ -191,6 +192,10 @@ type Guest struct {
 	// +optional
 	Ports []Port `json:"ports,omitempty"`
 
+	// Logical clock value corresponding to the desired resources of the VM.
+	// +optional
+	DesiredLogicalTime *LogicalTime `json:"desiredLogicalTime,omitempty"`
+
```

Comment:
IMO this should be a field on `VirtualMachineSpec` not `Guest` — `Guest` is more about the virtual machine itself; `spec` tends to be higher-level stuff relating to the VM object.

WDYT?